task3/vector.js

```diff
@@ -0,0 +1,81 @@
+function sortAsc(array) {
+    return array.sort((a,b) => a-b);
+}
+
+class Vector {
+    constructor(inputArray) {
+        this.array = inputArray;
+    }
+
+    checkIsSameLength(vector) {
+        if (this.array.length !== vector.array.length) {
+            throw "Two vectors should have a same length";
+        }
+    }
+
+    add(vector) {
+        this.checkIsSameLength(vector);
+        const result = [];
+        for (let index = 0; index < this.array.length; index++) {
+            result.push(this.array[index] + vector.array[index]);
+        }
+        return result;
```

Comment:
First of all, from this method we should get `new Vector([4, 6, 8])`, not just array, so you should return `new Vector(result)`.
And also you could improve this method by using `Array.map()`. It's a really helpful method :)
```
return new Vector(this.array.map((num, index) => {
    return num + array[index];
}))
```